packages/api/lambdas/db-indexer.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const get = require('lodash.get');
+const pLimit = require('p-limit');
+const { AttributeValue } = require('dynamodb-data-types');
+const indexer = require('../es/indexer');
+const { Search } = require('../es/search');
+const unwrap = AttributeValue.unwrap;
+
+function indexRecord(esClient, record) {
+  // only process if the source is dynamoDB
+  if (record.eventSource !== 'aws:dynamodb') {
+    return Promise.resolve();
+  }
+
+  let stack = process.env.stackName;
```

Comment:
This is not reassigned later on and should be a `const`

Reply:
https://github.com/cumulus-nasa/cumulus/pull/234/commits/05c31a6985b1a51ff2c45404c94665dfb34ee369